### PR TITLE
chore: Update release.yml

### PR DESCRIPTION
**Pull request by morshoto**
- [ ] Request PR review
- [ ] If you are unable to review, please set it as a [PR draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/) draft.
- [ ] Github issue: Closes: #

## Validation

```bash
pnpm run build
pnpm run test
pnpm run check:boundaries
```

Native changes should also include the relevant macOS/Xcode validation.

## Architecture and compatibility

- [ ] Runtime remains independent of MCP and editor-specific implementations.
- [ ] Public MCP behavior or package interfaces are documented if changed.
- [ ] Existing adapters and test fixtures remain compatible, or the migration is explained above.

## Safety

- [ ] No credentials, private media, raw crash dumps, or user-specific local paths were added.
- [ ] Generated files and build artifacts are excluded.
- [ ] Documentation reflects any changed commands, paths, or environment variables.

## Summary by CodeRabbit

- **Chores**
  - Disabled the release-runner preflight check in the release workflow.
  - Native release assets are now packaged without requiring the preflight validation step.
  - Other release workflow jobs and behavior remain unchanged.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,479 +1,478 @@
...
+              shell: bash
+              run: |
+                  set -euo pipefail
+                  if [[ "${RELEASE_TAG}" =~ ^v[0-9]+\.[0-9]+\.0$ ]]; then
+                    echo "milestone=true" >> "$GITHUB_OUTPUT"
+                  else
+                    echo "milestone=false" >> "$GITHUB_OUTPUT"
+                  fi
+
+    # release-runner-preflight:
```
[the remark this answers is not in the document]
Comment (morshoto) [reply .github/workflows/release.yml]: Restored the release-runner-preflight job and its dependency from native-release-assets in d5bbd290. The preflight still checks for an online, idle runner with self-hosted, macOS, and framekit-release labels via scripts/check-release-runner.mjs. Focused release-workflow tests pass 16/16; full validation also passes: build, 904 tests (903 passed, 1 skipped), and boundary checks.
